draft-ietf-rats-corim.md

```diff
@@ -484,11 +485,11 @@ The following describes each child item of this group.
 
 # Concise Module Identifier (CoMID) {#sec-comid}
 
-A CoMID tag contains information about hardware, firmware, or module composition.
+A CoMID tag contains information about subjects such as hardware, firmware, or module composition.
 
 Each CoMID has a unique ID that is used to unambigously identify CoMID instances when cross referencing CoMID tags, for example in typed link relations, or in a CoBOM tag.
 
-A CoMID defines several types of Claims, using "triples" semantics.
+A CoMID represents expectations for Evidence appraisal with "triples".
```

Comment:
Is the use of `expectations` trying to combine the meanings of reference values and endorsed values?